e2e/setups.go

```diff
@@ -66,7 +66,8 @@ func SetupDefaults(l *setupList) {
 							WithProtocol(httpbinScheme)).
 					AddService(
 						forwarder.ProxyService().
-							WithProtocol(proxyScheme)).
+							WithProtocol(proxyScheme).
+							WithGoleak()).
```

Comment:
I wonder, why not enable it by default?

Reply:
Right. I enabled goleak in a proxy service by default.